x-pack/platform/plugins/shared/fleet/server/saved_objects/index.ts

```diff
@@ -1237,4 +1296,9 @@ export function registerEncryptedSavedObjects(
     attributesToEncrypt: new Set(['token']),
     attributesToIncludeInAAD: new Set(['policy_id', 'token_plain']),
   });
+  encryptedSavedObjects.registerType({
+    type: FLEET_SERVER_HOST_SAVED_OBJECT_TYPE,
+    attributesToEncrypt: FLEET_SERVER_HOST_ENCRYPTED_FIELDS,
+    enforceRandomId: false,
```

Comment:
The reason why I explicitly added this line is that our `POST` endpoint supports the creation of an explicit id by the user and having an UUID instead would be a major breaking change.

Reply:
__question:__ does it mean you're making the `ssl` property, which was previously unencrypted, encrypted? Or am I missing something? If so, what about _existing_ objects where `ssl` property is stored in non-encrypted form? How would that work right now?

> The reason why I explicitly added this line is that our POST endpoint supports the creation of an explicit id by the user and having an UUID instead would be a major breaking change.

It'd be great if you could leave a comment in the code explaining this to help future readers.

Reply:
> what about existing objects where ssl property is stored in non-encrypted form?

That shouldn't be a problem as the `ssl` property is a new one added on the existing `FLEET_SERVER_HOST_SAVED_OBJECT_TYPE` SO. @elena-shostak in [this comment](https://github.com/elastic/kibana/pull/208091#discussion_r1933760450) suggested to do so as the `ssl` property will store sensitive information.

> It'd be great if you could leave a comment in the code explaining this to help future readers.

Sure, I'll add a comment there

Reply:
Got it, it wasn't clear whether it's a new or existing property, but if it's a new property, then we should be fine indeed. Thanks for confirming.